docs/hello-world.md

```diff
@@ -7,18 +7,18 @@ next: docs/development.md
 A Probot app is just a [Node.js module](https://nodejs.org/api/modules.html) that exports a function:
 
 ```js
-module.exports = robot => {
+module.exports = app => {
   // your code here
 }
 ```
 
-The `robot` parameter is an instance of [`Robot`](https://probot.github.io/api/latest/Robot.html) and gives you access to all of the bot goodness.
+The `app` parameter is an instance of [`Application`](https://probot.github.io/api/latest/Application.html) and gives you access to all of the GitHub goodness.
```

Comment:
nit: maybe Github goodness -> GitHub App goodness

Reply:
Since "GitHub Apps" is just the installation/authentication mechanism, I think I'm going to leave this for now.